app/validation/user.py

```diff
@@ -19,5 +19,5 @@ def is_respondent(self):
 
     @staticmethod
     def is_valid_user(uuid):
-        response = party.get_user_details(uuid)
-        return response.status_code == 200
+        response, status_code = party.get_user_details(uuid)
+        return status_code == 200
```

Comment:
I don't understand the point of returning the response and status code separately if the status code is included in the response anyway. You're just returning the same thing twice